pkg/inject/sidecar.go

```diff
@@ -278,6 +279,29 @@ func container(jaeger *v1.Jaeger, dep *appsv1.Deployment) corev1.Container {
 		SecurityContext: jaeger.Spec.Agent.SidecarSecurityContext,
 		VolumeMounts:    volumesAndMountsSpec.VolumeMounts,
 	}
+	if IsJaegerPortAvailable(adminPort, dep) {
```

Comment:
A better name could probably be `ContainerPort` instead of `JaegerPort`.

Reply:
Agree 100%